atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -149,6 +149,15 @@ public Properties getHikariProperties() {
         return new Properties();
     }
 
+    /**
+     * Please refer to <a href="https://github.com/brettwooldridge/HikariCP#infrequently-used">HikariCP</a>
+     * before overriding.
+     * */
+    @Value.Default
+    public long initializeFailTimeoutMillis() {
+        return 1;
```

Comment:
Could we make this an optional and then only set the timeout in Hikari if an override is present? Just so we don't have to make sure that this config default and the Hikari default match